docs/resources/samples/copy-tables-combine.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Copy Excel tables into new master table'
+description: 'Learn how to use Office Scripts to combine data from multiple Excel tables into a single table.'
+ms.date: 03/04/2021
+localization_priority: Normal
+---
+
+# Copy Excel tables into new master table
+
+Shows how to combine data that resides in Excel tables into a new master table with all rows included. It creates a new worksheet and combines all rows into a single table.
```

Comment:
Suggestion to make that first sentence more introductory. I'm also not sure that second sentence adds much.
```suggestion
This sample combines data from multiple Excel tables into a single table that includes all the rows.
```